src/Kunstmaan/MediaBundle/Helper/MediaManager.php

```diff
@@ -2,25 +2,80 @@
 
 namespace Kunstmaan\MediaBundle\Helper;
 
+use Exception;
 use Kunstmaan\MediaBundle\Entity\Media;
 use Kunstmaan\MediaBundle\Helper\Media\AbstractMediaHandler;
+use Kunstmaan\MediaBundle\Model\ImageUploadModel;
 use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\Mime\MimeTypesInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 /**
  * MediaManager
  */
 class MediaManager
 {
+    private const ALLOWED_EXTENSIONS = [
```

Comment:
I would make both lists of extension configurable through the media bundle config and inject them in the constructor. We can keep the constants (but rename them to `DEFAULT_ALLOWED_EXTENSIONS`?) here but set these arrays ase the default values in the config. This way a config-reference dump will also show these values.

This will give the enduser control over the allowed extensions